account/manager.go

```diff
@@ -624,9 +702,20 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 				}
 			}
 
-			acctKey := account.TraderKey.PubKey.SerializeCompressed()
-			contextLabel := fmt.Sprintf(" poold -- "+
-				"AccountCreation(acct_key=%x)", acctKey)
+			txFee, err := m.deriveFeeFromTx(ctx, accountTx)
+			fee := &txFee
+			if err != nil {
+				log.Errorf(
+					"Failed to derive fee from "+
+						"transaction: %v", accountTx,
+				)
+				fee = nil
+			}
+			balanceDiff := account.Value
+			contextLabel := actionTxLabel(
+				account, CREATE, false, fee,
```

Comment:
nit: I think `balanceDiff` should also fit on the previous line.

Reply:
`eDiff,` would be over the 80 char column (according to my IDE). Is that acceptable? Asking because I noticed that some lines within the codebase do go over.

I see that the guide asks for a "best effort basis": https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#80-character-line-length

Reply:
Hmm, weird. Then maybe some setting isn't right in your IDE perhaps?
This is what it looks in my IDE after merging L716 and L717 (visual aid set at 80 characters).
![Screenshot from 2022-11-04 19-42-53](https://user-images.githubusercontent.com/1008879/200051841-27dbb32d-6f95-4166-9bbf-c5481e293862.png)


Yes, it's best effort and we usually don't make a big deal out of a few characters.

Reply:
My mistake. I had inlay parameter hints turned on.